CRM/Event/Form/Task/Batch.php

```diff
@@ -268,7 +268,7 @@ public static function updatePendingOnlineContribution($participantId, $statusId
     $contributionId = CRM_Contribute_BAO_Contribution::checkOnlinePendingContribution($participantId,
       'Event'
     );
-    if (!$contributionId || !$participantId) {
```

Comment:
@eileenmcnaughton I thought we had just added this?

Reply:
@seamuslee001 yeah - well moved it really - I moved it from one place to another because I thought elinimating it in that PR would be too much

Reply:
What would happen if $participantId is not set in the updateContributionStatus function?

Reply:
it's not possible because the only function that calls it it iterating through an array of records keyed by participant id